ddtrace/ext/ci.py

```diff
@@ -218,19 +265,22 @@ def extract_gitlab(env):
         PIPELINE_URL: url,
         PROVIDER_NAME: "gitlab",
         WORKSPACE_PATH: env.get("CI_PROJECT_DIR"),
+        git.COMMIT_MESSAGE: env.get("CI_COMMIT_MESSAGE"),
     }
 
 
 def extract_jenkins(env):
-    branch_or_tag = env.get("GIT_BRANCH")
-    branch = tag = None
+    # type: (MutableMapping[str, str]) -> MutableMapping[str, Optional[str]]
+    """Extract CI tags from Jenkins environ."""
+    branch_or_tag = env.get("GIT_BRANCH", "")
+    branch = tag = None  # type: Optional[str]
     if "tags/" in branch_or_tag:
         tag = branch_or_tag
     else:
         branch = branch_or_tag
     name = env.get("JOB_NAME")
     if name and branch:
-        name = re.sub("/" + _normalize_ref(branch), "", name)
+        name = re.sub("/" + _normalize_ref(branch), "", name)  # type: ignore[operator]
```

Comment:
ignore operation is for `"/" + _normalize_ref(branch)`?

could do it as: `"/{0}".format(_normalize_ref(branch))` ?